### PR TITLE
Rename `requestCompleted` to `requesterCompletionStatus`

### PR DESCRIPTION
**Pull request by msasikanth**

### CODE REVIEW

**Comment by msasikanth** [comment]
Marking this as WIP, forgot to add migration

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,7 +5,8 @@
 - Update `TeleconsultRecord` to sync frequently
 - Add `TeleconsultStatus` to `TeleconsultRecord`
 - Mark strings used in prescription image as un-translatable
-- Use `US` locale when formatting prescription date
+- Use `ENGLISH` locale when formatting prescription date
```
Comment (Honey14) [inline CHANGELOG.md:8]: Can you explain why are we changing the locale here?

**Review by Honey14 — CHANGES_REQUESTED** [review]
Looks good. Left few comments. PTAL

File: CHANGELOG.md (hunk 1 above)
Comment (msasikanth) [reply CHANGELOG.md:8]: We are not actually changing it in this PR, it's just a typo fix in the CHANGELOG from #2021

File: app/src/sharedTest/java/org/simple/clinic/TestData.kt [hunk 2]
```diff
@@ -1211,7 +1211,7 @@ object TestData {
         requesterId = requesterId,
         facilityId = facilityId,
         requestedAt = requestedAt,
-        requestCompleted = requestCompleted
+        requesterCompletionStatus = requestCompleted
```
[the remark this answers is not in the document]
Comment (msasikanth) [reply app/src/sharedTest/java/org/simple/clinic/TestData.kt:1214]: Ah good catch, will fix it

**Review by Honey14 — APPROVED** [review]
Looks good

**Pull request merged**